plugins/reports/monographReport/MonographReportPlugin.php

```diff
@@ -82,76 +100,352 @@ public function display($args, $request): void
+            Decision::INITIAL_DECLINE => __('editor.submission.decision.decline'),
+            Decision::RECOMMEND_ACCEPT => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.accept')]),
+            Decision::RECOMMEND_PENDING_REVISIONS => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.requestRevisions')]),
+            Decision::RECOMMEND_RESUBMIT => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.resubmit')]),
+            Decision::RECOMMEND_DECLINE => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.decline')]),
+            Decision::RECOMMEND_EXTERNAL_REVIEW => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.sendExternalReview')]),
+            Decision::NEW_EXTERNAL_ROUND => __('editor.submission.decision.newReviewRound'),
+            Decision::REVERT_DECLINE => __('editor.submission.decision.revertDecline'),
+            Decision::REVERT_INITIAL_DECLINE => __('editor.submission.decision.revertDecline'),
+            Decision::SKIP_EXTERNAL_REVIEW => __('editor.submission.decision.skipReview'),
+            Decision::SKIP_INTERNAL_REVIEW => __('editor.submission.decision.skipReview'),
+            Decision::ACCEPT_INTERNAL => __('editor.submission.decision.accept'),
+            Decision::PENDING_REVISIONS_INTERNAL => __('editor.submission.decision.requestRevisions'),
+            Decision::RESUBMIT_INTERNAL => __('editor.submission.decision.resubmit'),
+            Decision::DECLINE_INTERNAL => __('editor.submission.decision.decline'),
+            Decision::RECOMMEND_ACCEPT_INTERNAL => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.accept')]),
+            Decision::RECOMMEND_PENDING_REVISIONS_INTERNAL => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.requestRevisions')]),
+            Decision::RECOMMEND_RESUBMIT_INTERNAL => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.resubmit')]),
+            Decision::RECOMMEND_DECLINE_INTERNAL => __('editor.submission.recommendation.display', ['recommendation' => __('editor.submission.decision.decline')]),
+            Decision::REVERT_INTERNAL_DECLINE => __('editor.submission.decision.decline'),
+            Decision::NEW_INTERNAL_ROUND => __('editor.submission.decision.newReviewRound'),
+            Decision::BACK_FROM_PRODUCTION => __('editor.submission.decision.backToCopyediting'),
+            Decision::BACK_FROM_COPYEDITING => __('editor.submission.decision.backFromCopyediting'),
+            Decision::CANCEL_REVIEW_ROUND => __('editor.submission.decision.cancelReviewRound'),
+            Decision::CANCEL_INTERNAL_REVIEW_ROUND => __('editor.submission.decision.cancelReviewRound'),
```

Comment:
All these decisions are not covered at the original plugin: https://github.com/pkp/ojs/blob/5d4d6c3f9fa324aef29a085d9179ef1e74ae7bc4/plugins/reports/articles/ArticleReportPlugin.php#L337-L365

Some are marked as "OMP only", but others not... I didn't inspect deep enough to recognize if it's a bug.

Reply:
Ok 😁